cmd/osmosisd/cmd/root.go

```diff
@@ -99,17 +96,32 @@ type DenomUnitMap struct {
 	Exponent uint64 `json:"exponent"`
 }
 
-const (
-	mempoolConfigName = "osmosis-mempool"
-
-	arbitrageMinGasFeeConfigName          = "arbitrage-min-gas-fee"
-	recommendedNewArbitrageMinGasFeeValue = "0.1"
-
-	maxGasWantedPerTxName                = "max-gas-wanted-per-tx"
-	recommendedNewMaxGasWantedPerTxValue = "60000000"
+type SectionKeyValue struct {
+	Section string
+	Key     string
+	Value   any
+}
 
-	consensusConfigName     = "consensus"
-	timeoutCommitConfigName = "timeout_commit"
+var (
+	recommendedAppTomlValues = []SectionKeyValue{
+		{
+			Section: "osmosis-mempool",
+			Key:     "arbitrage-min-gas-fee",
+			Value:   "0.1",
+		},
+		{
+			Section: "osmosis-mempool",
+			Key:     "max-gas-wanted-per-tx",
+			Value:   "60000000",
+		}}
+
+	recommendedConfigTomlValues = []SectionKeyValue{
+		{
+			Section: "consensus",
+			Key:     "timeout_commit",
+			Value:   "2s",
+		},
+	}
```

Comment:
With this abstraction, for all future changes all we need to do is add to this var, and all the rest is taken care of!